test/visual-regression/index.js

```diff
@@ -89,6 +89,7 @@ function beginApplication(browser) {
 
       // Tell Nighwatch we're finished.
       .then(done);
+    // .catch(err => console.log(err.stack));
   });
 }
 
```

Comment:
We can probably remove the changes to the VRT files since that was just from when we were debugging a couple weeks ago